packages/components/copybutton/src/CopyButton.tsx

```diff
@@ -134,6 +139,8 @@ function _CopyButton(
       ref={ref}
     />
   );
+
+  // ToDo reimplement tooltip
   // return (
```

Comment:
Can we add the files with these Todo's to the tooltip ticket? so we don't forget any.